src/main/java/life/qbic/cli/ToolExecutor.java

```diff
@@ -73,40 +77,49 @@ protected AbstractCommand validateParametersAndParseCommandlineArguments(final C
     return AbstractCommand.parseArguments(commandClass, args);
   }
 
-  private Tool instantiateTool(final Class<? extends QBiCTool> toolClass, final AbstractCommand command) {
+  private Tool instantiateTool(final Class<? extends QBiCTool> toolClass,
+      final AbstractCommand command) {
     final Tool tool;
     final Constructor<? extends QBiCTool> constructor;
 
     try {
       constructor = toolClass.getConstructor(command.getClass());
     } catch (final NoSuchMethodException e) {
       throw new ApplicationException(String
-          .format("Could not find a suitable public constructor for the given tool with class name %s. Check QBiCTool class' documentation.", toolClass),
+          .format(
+              "Could not find a suitable public constructor for the given tool with class name %s. Check QBiCTool class' documentation.",
+              toolClass),
           e);
     }
 
     try {
       tool = constructor.newInstance(command);
     } catch (final InstantiationException | IllegalAccessException | InvocationTargetException e) {
-      throw new ApplicationException(String.format("Could not create a new instance for the given tool with class name %s", toolClass), e);
+      throw new ApplicationException(
+          String.format("Could not create a new instance for the given tool with class name %s",
+              toolClass), e);
     }
 
     return tool;
   }
 
   /**
-   * Handles the standard {@ code --help} and {@code --version} parameters which all {@link QBiCTool} are able to handle.
+   * Handles the standard {@ code --help} and {@code --version} parameters which all {@link
+   * QBiCTool} are able to handle.
    *
    * @param toolMetadata the metadata of the tool.
-   * @param command the command.
-   * @return {@code true} if either help or version (or both!) were requested, {@code false} otherwise.
+   * @param command      the command.
+   * @return {@code true} if either help or version (or both!) were requested, {@code false}
+   * otherwise.
    */
-  protected boolean handleCommonParameters(final ToolMetadata toolMetadata, final AbstractCommand command) {
+  protected boolean handleCommonParameters(final ToolMetadata toolMetadata,
+      final AbstractCommand command) {
```

Comment:
I'm confused about the spacing adaptions in this class? Is this formatted as intended?

Reply:
might have been something to do with the eclipse codestyle, if it was changed by me. I don't remember unfortunately

Reply:
I think this formatting is correct as it is a continuation of the previous line